src/codeflare_sdk/ray/rayjobs/rayjob.py

```diff
@@ -282,3 +299,265 @@ def _map_to_codeflare_status(
         return status_mapping.get(
             deployment_status, (CodeflareRayJobStatus.UNKNOWN, False)
         )
+
+    def _extract_script_files_from_entrypoint(self) -> Optional[Dict[str, str]]:
+        """
+        Extract local Python script files from entrypoint command, plus their dependencies.
+
+        Returns:
+            Dict of {script_name: script_content} if local scripts found, None otherwise
+        """
+        if not self.entrypoint:
+            return None
+
+        scripts = {}
+        mount_path = "/home/ray/scripts"
```

Comment:
nit: could we make this a constant somewhere since it's referenced in a good few places and if we for some reason need / want to change it in the future, it'd be easier if it was a constant. Not necessarily in a constants file but at the top level of the class or something (whatever works)